pom.xml

```diff
@@ -34,18 +34,17 @@
     </issueManagement>
 
     <properties>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <revision>2.6.5</revision>
+        <changelist>-SNAPSHOT</changelist>
         <jenkins.version>2.332.3</jenkins.version>
-        <spotbugs.effort>Default</spotbugs.effort>
-        <spotbugs.failOnError>true</spotbugs.failOnError>
-        <spotbugs.threshold>Medium</spotbugs.threshold>
+        <gitHubRepo>jenkinsci/active-choices-plugin</gitHubRepo>
     </properties>
 
     <scm>
-        <connection>scm:git:git://github.com/jenkinsci/active-choices-plugin.git</connection>
-        <developerConnection>scm:git:git@github.com:jenkinsci/active-choices-plugin.git</developerConnection>
-        <url>https://github.com/jenkinsci/active-choices-plugin</url>
-        <tag>HEAD</tag>
+        <connection>scm:git:https://github.com/${gitHubRepo}.git</connection>
+        <developerConnection>scm:git:git@github.com:${gitHubRepo}.git</developerConnection>
+        <url>https://github.com/${gitHubRepo}</url>
+        <tag>${scmTag}</tag>
```

Comment:
The result of applying this [best practice](https://www.jenkins.io/doc/developer/tutorial-improve/enable-incrementals/).